src/main/java/com/google/cloud/tools/managedcloudsdk/process/CollectingByteHandler.java

```diff
@@ -17,16 +17,13 @@
 package com.google.cloud.tools.managedcloudsdk.process;
 
 /** A line handler that collects lines and returns a single result string. */
-public class CollectingLineHandler implements LineHandler<String> {
+public class CollectingByteHandler implements ByteHandler<String> {
```

Comment:
What would CollectingByteHandler be used for? It seems the same can be accomplished with just a MessageListenerForwardingHandler.

Reply:
It's for getting results from a command like when testing if gcloud has a component or something. It's just a convenience mechanism at this point, but it's value will be evident in a followup CL.